cmake/Finddpdk.cmake

```diff
@@ -84,6 +84,14 @@ endif ()
 list (APPEND dpdk_REQUIRED
   dpdk_INCLUDE_DIR)
 
+# Depending on whether libbsd-dev exists at build time, DPDK may be built with a dependency on
+# libbsd. Note that other libraries in dpdk_PC_LIBRARIES are handled using separate logic
+# (see rte_libs above), thus the additional dependencies must be handled on a case by case basis.
+set (dpdk_dependencies "")
```

Comment:
no need to initialize `dpdk_dependencies` . after we `list(APPEND v "foo")` when `v` is not initialized at all, the value of `v` is `"foo"`.

Reply:
Fixed now.